src/test/java/create_task_use_case/CreateTaskOutputDataTest.java

```diff
@@ -0,0 +1,45 @@
+package create_task_use_case;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+/**
+ * Task Creation Output Data Test Cases.
+ * @author Fardin Faruk
+ */
+public class CreateTaskOutputDataTest {
+    @Test
+    public void testGetTitle(){
+        CreateTaskOutputData outputData = new CreateTaskOutputData("Feed Kittens", 0,
+                "");
+        assertEquals(outputData.getTitle(), "Feed Kittens");
```

Comment:
make sure you use consistent order of parameters in assertEquals statements.

Reply:
done